examples/routes/simple-counter.js

```diff
@@ -0,0 +1,12 @@
+let counter = 0;
+
+module.exports = function (addRoute) {
+	addRoute(2020, {
+		uri: '/sqren',
+		method: 'GET',
+		handler: function (req, res) {
+			counter++;
```

Comment:
if handler function is executed on mock server, how is it possible to reference a variable outside of this function?